django_tasks/backends/database/admin.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib import admin
+
+from .models import DBTaskResult
+
+
+@admin.register(DBTaskResult)
+class DBTaskResultAdmin(admin.ModelAdmin):
+    list_display = ("id", "task_path", "status", "priority", "queue_name")
```

Comment:
Suggestion: It might be easier to read to use the task's `.name` attribute, as it'll be more human-readable than the full path (not to mention much shorter)

```suggestion
    list_display = ("id", "task_path", "status", "priority", "queue_name")
```